.github/workflows/functional-tests.yml

```diff
@@ -0,0 +1,78 @@
+name: "Functional Test"
+
+on:
+  push:
+  pull_request:
+
+env:
+  SIMULATION: native
+  ENABLE_UNIT_TESTS: true
```

Comment:
Is there a way to build the test app without building all the unit tests?  Building all the unit tests seems like overkill.

Reply:
no